tomcat-gcp-lib/src/main/java/com/google/cloud/runtimes/tomcat/logging/HttpLabels.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.runtimes.tomcat.logging;
```

Comment:
@cassand I just noticed that this class is in the `logging` package. Why not `trace`?
[This might be an issue to deal with in a different PR though]

Reply:
That would indeed make sense to refactor later on. It would have impact on the other modules in this project.

Reply:
This was initially in reference to Stackdriver Logging, I will make the change once this PR is merged.

Here is the associated issue: #90